### ISSUE TITLE
Graphic design for the Visualization tab under the Ecosystem section

### ISSUE THREAD

**Issue by ghost** [numpy/numpy.org#485]
This issue is raised in #342 by @InessaPawson.

I am just making a specific issue out of it so that people give attention to it. 

Ecosystem section comes on the home page, and Science Tab appears even first when you visit the Ecosystem Section, but there are other Tabs too which have not been given enough attention like Data Science and Visualization.

@InessaPawson you want new images for the Visualization Tab or want to modify the existing ones ?

**Comment by InessaPawson** [comment]
> @InessaPawson
> 
> What i suggested was not to remove the "Visualization" section, I suggested to add a page to the NumPy website which explains the related things in detail.
> 
> Which might ultimately lead to the better development of that Tab.
> 
> Do u think it is under-developed too ?

@new-vivek11 The objective of the Visualization tab was to underline the importance of NumPy in this space by showcasing some of the most popular and highly reputable visualization libraries that depend on NumPy. With that in mind, I think the amount of the existing content will suffice.

**Comment by rgommers** [comment]
> With that in mind, I think the amount of the existing content will suffice.

I agree. We want to give a sketch of the ecosystem, but not explain everything in a lot of detail. That would be a ton of work, and would also give a lot more content to translate into a number of other languages. So while we can tweak things if needed, in general we're happy with the amount of content on the site right now, and the quality of it.

For visualization in more detail, please see https://pyviz.org/
